src/components/EmptyState/EmptyState.scss

```diff
@@ -18,7 +18,6 @@
 
         &_size_m {
             position: relative;
-            top: 20%;
```

Comment:
Because of this value content sometimes didn't fit in wrapper box and was partly hidden